tests/object/test_common.py

```diff
@@ -6,6 +7,9 @@
 from typing import Tuple
 import pytest
 
+from unittest import mock
```

Comment:
are those imports required? I don't see the usage of them

Reply:
good point, removed